t/venom/test_suites/pfcron/01_job_was_ran.yml

```diff
@@ -6,15 +6,24 @@ testcases:
   - type: exec
     script: 'systemctl stop packetfence-pfcron'
 
+  - type: exec
+    script: 'cp "{{.fingerbank_upstream_db}}" "{{.fingerbank_upstream_db}}.bak"'
+
   - type: exec
     script: 'cp "{{.pfserver_root_dir}}/t/data/pfcron-test.conf.test" "{{.pfserver_conf_dir}}/pfcron.conf"'
 
+  - type: exec
+    script: '{{.ini_cmd}} {{.fingerbank_conf_dir}}/fingerbank.conf add upstream host 127.0.0.1 upstream port 3000 upstream use_https disabled'
```

Comment:
I don't see where you defined that mock server is listening on 127.0.0.1:3000

Reply:
That is the default port of the mock server however it does make sense to make it a variable.